mat-process-api.Tests/V1/Controllers/ProcessDataControllerTests.cs

```diff
@@ -128,7 +128,7 @@ public void given_GetProcessData_controller_method_call_when_request_validation_
         }
 
         [Test]
-        public void test_that_controller_returns_correct_message_and_status_code_when_no_document_was_found()
+        public void given_DocumentNotFound_exception_is_thrown_when_GetProcessData_controller_method_returns_404_status_code_with_correct_error_message()
```

Comment:
There's an accidentally deleted bit in the test name:
`when_GetProcessData_controller_[deleted bit]_method_returns`